FrontEnd/src/components/AboutPage/MeetTheTeam/MemberInfoCard/MemberInfoCard.module.scss

```diff
@@ -0,0 +1,77 @@
+.card {
+  width: 22.125rem;
+  height: 11.125rem;
+  border-radius: 0.5rem;
+  overflow: hidden;
+  border: 0.25rem solid #f8e6d8 !important;
+}
+
+.firstCol {
+  width: 17.625rem;
+  height: 11rem;
+
+  display: flex;
+  order: 0;
+  flex-direction: column;
+
+  border-right: 0.5rem solid #f8e6d8;
+  box-sizing: border-box;
+}
+
+.information {
+  padding: 1.5rem 0.5rem 1.5rem 2.25rem;
+}
+
+.nameinfo {
+  flex: display;
+  margin: 0rem 0.5rem;
+  align-self: center;
+  letter-spacing: 0.03em;
+
+  .name {
+    color: #000000;
+  }
+
+  .title {
+    color: rgba(32, 32, 32, 0.6);
+  }
+}
+
+.caption {
+  color: rgba(32, 32, 32, 0.87);
```

Comment:
Same as the other color comment. We want to avoid magic numbers like this so it would be easier to update in the future if necessary

The same logic applies to the rest of the colors in this file too. I didn't add comments on each of them because that's a lot of repetitive comments